JustTrack/JEEventsGenerator/main.swift

```diff
@@ -348,7 +358,7 @@ do {
     //write struct file
     let structSwiftFilePath = CommandLine.arguments[2]
     if !FileManager.default.fileExists(atPath: structSwiftFilePath) {
-        throw JEStructsGeneratorError.swiftFileNotFound
+        FileManager.default.createFile(atPath: structSwiftFilePath, contents: .none, attributes: .none)
```

Comment:
the main issue here is that if the script generates the file that file isn't added to the project, and could be really difficult for the developer to understand what's gone wrong. A build failure is much more direct and clear.